app/server.js

```
@@ -250,6 +254,29 @@ io.on("connection", (socket) => {
     argjson.DateCreated = new Date().toISOString()
     
+    if(argjson.method === "modern") {
+      const session = await auth.api.getSession({
+        headers: new Headers({
+          authorization: "Bearer " + argjson.token
+        })
+      });
+      if(argjson.parentmessageid) {
+        var parentid = argjson.parentmessageid
+      } else {
+        var parentid = null
```

Biome (1.9.4)

[error] Shouldn't redeclare 'parentid'. Consider to delete it or rename it.

'parentid' is defined here:


(lint/suspicious/noRedeclare)